### PR TITLE
Provides Adviser import capabilities

### PR DESCRIPTION
**Pull request by benlovell**
* Adds `Lookup::Adviser` and basic validation
* `import:advisers` rake task

I'll squash ddc04ce and 7548daf once I get the thumbs :+1:

### CODE REVIEW

File: app/models/lookup/adviser.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module Lookup
+  class Adviser < ActiveRecord::Base
+    validates :reference_number, length: { is: 8 }
+
+    validates_presence_of :name
```
Comment (tomas-stefano) [inline app/models/lookup/adviser.rb]: Curiosity, :). Why in one validation you use the options format and here you called the method name? Why did not do `validates :name, presence: true'?

Comment (benlovell) [inline app/models/lookup/adviser.rb]: Good point. I added another commit to tidy this up.

**Comment by tomas-stefano** [comment]
:+1:

**Comment by lshepstone** [comment]
:+1:

**Pull request merged**